infra/proposer.go

```diff
@@ -11,32 +11,38 @@ import (
 )
 
 type Proposers struct {
-	workers []*Proposer
+	workers [][]*Proposer
```

Comment:
not a bit deal but would it be simpler if we start multiple `Proposers` from `main` for each peer and keep this struct intact?

Reply:
Skip for now.

Reply:
why skip? code looks cleaner (no nested for loop))

Reply:
```
type Proposers struct {
	workers [][]*Proposer
type Proposer struct {
	e    peer.EndorserClient
```

in my point of view,   ``` workers [number of endorsement clients][number of working number ]*Proposer```

pls let me know, 
```
Proposers[]//numbers of ?
type Proposers struct {
	workers []*Proposer //numbers of ?
type Proposer struct {
	e    peer.EndorserClient
```